app/src/ui/lib/app-proxy.ts

```diff
@@ -21,6 +20,7 @@ export type PathType =
   | 'recent'
   | 'logs'
   | 'crashDumps'
+  | 'sessionData'
```

Comment:
Non-impacting change: This `PathType ` is consumed in `ipc-shared.ts` by the `'get-path'` response channel definition. This is used in `main-process-proxy.ts` by `getPath` method. The `getPath` method is used in only used in `app-proxy.ts` and only uses the types of `'home'` and `'documents'`. 

![image](https://user-images.githubusercontent.com/75402236/209685349-5e6808af-cd2a-4779-9bac-d988381d5eb3.png)